docs/deployment/managers.md

```diff
@@ -1,7 +1,63 @@
 # Platform managers
```

Comment:
You can start with a short introduction explaining what these managers are, and that the page will explain what each manager handles as an action.